packages/firebase_database/example/lib/main.dart

```diff
@@ -97,7 +97,7 @@ class _MyHomePageState extends State<MyHomePage> {
     _counterSubscription.cancel();
   }
 
-  Future<void> _increment() async {
+  Future<void> _incrementWithTransaction() async {
```

Comment:
What is the purpose of this new method? as it isn't called anywhere

Reply:
Thank you for the prompt answer on this,
My intention here was to update the example already in place because the ```update()``` method return void and it's meant to be used to get a value.